docs/validations.mdx

```diff
@@ -6,25 +6,54 @@ import * as ValidationsStories from "./validations.stories";
 
 # Getting started with validations, Formik and Yup
 
-### Contents
+## Contents
 
 - [Carbon validation overview](#carbon-validation-overview)
-  - [Inputs](#inputs)
-  - [Grouped inputs](#grouped-inputs)
+  - [Validation redesign](#validation-redesign)
+  - [Validation states](#validation-states)
+  - [Input validation](#input-validation)
+  - [Grouped inputs validation](#grouped-inputs-validation)
   - [Validation icon placement](#validation-icon-placement)
 - [Formik overview](#formik-overview)
 
 ## Carbon validation overview
 
-Carbon provides inputs which are able to visually indicate three states of validations out of the box.
+Input components in Carbon have built-in validation states, which can be used to provide feedback to users about the state of their input. 
+These states can indicate whether the input is valid or not, and can also prevent form submission if the input is invalid.
 
-- **error**  -  meant to be a blocking validation, displays 2px wide red border and validation icon if message is provided.
-- **warning**  –  meant to be a non-blocking validation, displays 1px wide orange border and validation icon if message is provided.
-- **info**  –  meant to be a non-blocking validation, displays 1px wide blue border and validation icon if message is provided.
+### Validation redesign
 
-### Inputs
+Carbon is in the process of implementing a new validation pattern to our input components. To opt into this new pattern, you can set the 
+`validationRedesignOptIn` flag to true in the [CarbonProvider](https://carbon.sage.com/?path=/docs/carbon-provider--docs).
 
-Each input component which supports validations accepts following props - `error`, `warning` or `info`.
+Component props that are not supported if the opt-in flag is set to true are labelled as [Legacy] in their corresponding prop tables. 
+These props should not be used if the new validation pattern is being used.
+
+**Note:** This new pattern **does not** support inline labels/legends, this means that any related props will not have any effect or may have unexpected behaviour. 
+
+In components where the new validation pattern has not been implemented, the default validation pattern is still available, even if the opt in flag has been set to true.
+
+#### Validation redesign states
+
+- **error** - meant to be a blocking validation, styles the input with a 2px wide red border along with a red border on the left side of the component and a validation message if provided.
+- **warning** - meant to be a non-blocking validation, displays an orange border on the left side of the component and a validation message if provided.
+
+Input hints are used instead of Help tooltips to provide additional information about the input. A new `inputHint` prop has been added to support this, however, 
+in some components the `labelHelp` prop has been repurposed to function as an input hint. 
```

Comment:
suggestion: it might be worth adding something similar around this to the `labelHelp` stories like we've done with the `labelInline` ones. Happy for it to be done in a new ticket if you prefer just let me know

Reply:
I made a note of this in the validation stories for the components there this is the case, e.g.: [checkbox](https://5ecf782fe724630022d27d7d-qrkpwgjgny.chromatic.com/iframe.html?viewMode=docs&id=checkbox--docs#validations). Though I can add/move this to the `labelHelp` stories similar to `inlineLabel` if that would make it more visible/clear.

Reply:
I think you've missed some like [Textbox](https://github.com/Sage/carbon/pull/6777/files#diff-474a230e006f57f6a6c0ac0c2e4f8775876dc8a4dd1760517d257bdf8f75309e). I think we could do both, include it in the `labelHelp` and `new validation` stories. I'm happy to approve this for now as it wasn't part of the ticket but I think a new ticket should be raised to document the labelHelp nuances.